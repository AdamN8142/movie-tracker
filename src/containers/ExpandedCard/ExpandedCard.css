@import url('https://fonts.googleapis.com/css?family=Monoton');
@import url('https://fonts.googleapis.com/css?family=Josefin+Slab|Lobster|Montserrat|Playfair+Display');
.expanded-card {
  background-image: url('https://i.pinimg.com/originals/0e/86/98/0e86989645e59fd230f5f9f7bcf7eaff.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  display: flex;
  flex-direction: column;
  align-items: center;
  /* height: 100%; */
}

.expanded-photo { 
  width: 700px;
  height: 350px;
  border: 10px solid black;
}

.expanded-title {
  font-size: 50px;
  font-family: 'Montserrat', sans-serif;
  color: wheat;
}

.expanded-descrip {
  border:  2px solid black;
  /* margin-left: 90px;
  margin-right: 90px; */
  width: 500px;
  padding: 15px;
  font-size: 20px;
  background-color: white;
  opacity: .5;
}

.expanded-release {
  font-size: 30px;
  margin-top: 0;
  color: rgba(238, 53, 53, 0.815);
  
}