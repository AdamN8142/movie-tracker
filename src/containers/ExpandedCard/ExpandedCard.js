import React from 'react'
import './ExpandedCard.css'

export default function ExpandedCard({title, backdrop_path, release_date, overview}) {
  return (
    <div className='expanded-card'>
      <button>X</button>
      <img className="expanded-photo" alt={`${title}movie poster`} src={`https://image.tmdb.org/t/p/w500${backdrop_path}`} />
      <h1 className="expanded-title">{title}</h1>
      <h4 className="expanded-release">{release_date}</h4>
      <p className="expanded-descrip">{overview}</p>
    </div>
  )
}
