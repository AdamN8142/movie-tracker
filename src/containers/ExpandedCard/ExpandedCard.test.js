import React from 'react'
import {shallow} from 'enzyme'
import ExpandedCard from './ExpandedCard'


describe('Expandedcard', () => {
  let wrapper

  it('should match snapshot', () => {
    wrapper = shallow(<ExpandedCard/>)
    expect(wrapper).toMatchSnapshot()
  })
})