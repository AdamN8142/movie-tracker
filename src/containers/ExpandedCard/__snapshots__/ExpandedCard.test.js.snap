// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expandedcard should match snapshot 1`] = `ShallowWrapper {}`;
